asching7108/purrfect-match-api
# Pm 2 set up node express backend

Folders are separated between controllers, models, and utility functions. If you have and design changes, I am certainly open to them, so let me know!

If you want to test:

1. run `npm start` in root folder
2. Send a GET request to localhost:8000/pets
3. You should get a list of objects representing two pets

See project structure overview here:
https://purrfect-match-osu.atlassian.net/wiki/spaces/PM/pages/491564/API+Project+Structure
